pyogrio/tests/test_geopandas_io.py

```diff
@@ -1488,11 +1557,17 @@ def test_write_geometry_z_types_auto(
     assert_geodataframe_equal(gdf, result_gdf)
 
 
-def test_read_multisurface(data_dir):
-    df = read_dataframe(data_dir / "test_multisurface.gpkg")
+def test_read_multisurface(data_dir, use_arrow):
+    if use_arrow:
+        with pytest.raises(shapely.errors.GEOSException):
+            # TODO(Arrow)
+            # shapely fails parsing the WKB
+            read_dataframe(data_dir / "test_multisurface.gpkg", use_arrow=use_arrow)
+    else:
+        df = read_dataframe(data_dir / "test_multisurface.gpkg", use_arrow=use_arrow)
```

Comment:
```suggestion
        df = read_dataframe(data_dir / "test_multisurface.gpkg")
```

(dropping `use_arrow=False` here because it makes it a little harder to follow)